modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -756,6 +808,24 @@ The amount of memory to give an instance of a Data Transform (Wasm) virtual mach
 
 ---
 
+=== data_transforms_read_buffer_memory_percentage
+
+include::reference:partial$internal-use-property.adoc[]
```

Comment:
I think data loss is a little strong TBH

Reply:
Hi @rockwotj. To add a bit more context, the phrasing for this section was introduced here https://github.com/redpanda-data/docs/pull/479#discussion_r1591443806.

Reply:
@wzzzrd86 @mattschumpert  I recollect that this warning was what we came up with when discussing whether or not to document/publish the internal use only properties. The wording needs to be clear to adequately stave off users from "experimenting" with the internal use only props.